node/module_test.ts

```diff
@@ -15,7 +15,7 @@ const require = createRequire(import.meta.url);
 
 Deno.test("requireSuccess", function () {
   // Relative to import.meta.url
-  const result = require("./tests/cjs/cjs_a.js");
+  const result = require("./_module/cjs/cjs_a.js");
```

Comment:
Name this testdata instead for consistency?

Reply:
I named it that way since that folder contains the tests for `module.ts`, so I think it's more consistent this way (`stream` has `_stream`, `crypto` has `_crypto`, etc)